android/build.sh

```diff
@@ -16,13 +13,73 @@ THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
 . "${THIS_SCRIPT%/*}/../resources/build/build-utils.sh"
 ## END STANDARD BUILD SCRIPT INCLUDE
 
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+# This script runs from its own folder
+cd "$THIS_SCRIPT_PATH"
+
+################################ Main script ################################
+
+builder_describe \
+  "Build Keyman Engine for Android, Keyman for Android, and FirstVoices Android app." \
+  clean \
+  build \
+  "configure              Download asset .kmp files from downloads.keyman.com" \
+  "publish                Publishes the APKs to the Play Store." \
+  ":app                   Keyman for Android" \
+  ":engine                Keyman Engine for Android" \
+  ":samples               Sample apps: KMSample1 and KMSample2" \
+  ":keyboardharness       Test/KeyboardHarness app" \
```

Comment:
These can be defined as child projects, e.g.

```
":engine=KMEA    Keyman Engine for Android" \
```

(And in future when we rename the KMEA folder can drop `=KMEA`).

Then it will be built automatically with the `builder_run_child_actions` call.

Reply:
ic. Though `:samples` wouldn't handle both KMSample1 and KMSample2?

If so, then maybe I'd do `:kmsample1` and `:kmsample2`

Reply:
> If so, then maybe I'd do :kmsample1 and :kmsample2

Yes, either this or a separate samples build.sh.